packages/react-components/react-message-bar/library/src/components/MessageBar/useMessageBarReflow.ts

```diff
@@ -27,7 +27,8 @@ export function useMessageBarReflow(enabled: boolean = false) {
       }
 
       const entry = entries[0];
-      const borderBoxSize = entry?.borderBoxSize[0];
+      // `borderBoxSize` is not supported before Chrome 84, Firefox 92, nor Safari 15.4
+      const borderBoxSize = entry?.borderBoxSize?.[0];
```

Comment:
I think we can expand support by using `getBoundingClientRect()` when `borderBoxSize` is unavailable.

https://github.com/microsoft/fluentui-contrib/blob/main/packages/houdini-utils/src/paint/fallback/anims/play.ts#L28-L35